src/Sanitizer.ts

```diff
@@ -82,7 +83,7 @@ export class Sanitizer {
     /**
      * Canonicalize an email address.
      */
-    normalizeEmail(str: string, lowercase?: boolean): string {
+    normalizeEmail(str: string, lowercase?: boolean): string | false {
```

Comment:
The signature of `normalizeEmail` is as follow:
```ts
normalizeEmail(email: string, options?: NormalizeEmailOptions): string | false;
```

And if you look at [this](https://github.com/chriso/validator.js/blob/master/src/lib/normalizeEmail.js#L153) it indeeds returns false when it's not `isEmail`.

5.1 was different but still would return [false also](https://github.com/chriso/validator.js/blob/5.1.0/src/lib/normalizeEmail.js#L13)